components/slack/sources/new-interactions/new-interactions.mjs

```diff
@@ -0,0 +1,83 @@
+import common from "../common/base.mjs";
+
+export default {
+  name: "New Interactions",
+  version: "0.0.1",
+  key: "new-interactions",
+  description: "Emit new events on each new interactions",
+  type: "source",
+  props: {
+    ...common.props,
+    action_ids: {
+      type: "string[]",
+      label: "Action IDs",
+      description:
+        "A list of specific `action_id`'s to subscribe for new interaction events.",
```

Comment:
"...to subscribe to"

I'd maybe add a note about "if you don't select any Action IDs, then the source will subscribe to all action IDs"?

Reply:
Good point! I've added a better description to the `channels` prop too.